vsintegration/src/FSharp.Editor/LanguageService/LanguageService.fs

```diff
@@ -169,7 +165,6 @@ type internal FSharpWorkspaceServiceFactory [<Composition.ImportingConstructor>]
                                         nameof keepAllBackgroundSymbolUses, keepAllBackgroundSymbolUses
                                         nameof enableBackgroundItemKeyStoreAndSemanticClassification,
                                         enableBackgroundItemKeyStoreAndSemanticClassification
-                                        nameof captureIdentifiersWhenParsing, captureIdentifiersWhenParsing
```

Comment:
Or you could keep that telementry entry in there, with the value of `enableFastFindReferences`.
That is, if you want to make life easy for future people trying to make combined analysis of telemetry data before+after this PR.

Reply:
Yeah why not.